helm/hpcc/values.schema.json

```diff
@@ -393,10 +407,77 @@
         "detail": {
           "type": "integer",
           "description": "Log output verbosity"
+        },
```

Comment:
Indentation looks a bit odd here - tabs perhaps?

Reply:
Fixed, and added the githook to avoid tabs